openslides_backend/action/actions/motion/update.py

```diff
@@ -1,13 +1,23 @@
+import re
 import time
 from typing import Any, Dict
 
 from ....models.models import Motion
 from ....shared.exceptions import ActionException
-from ....shared.patterns import POSITIVE_NUMBER_REGEX, Collection, FullQualifiedId
+from ....shared.filters import FilterOperator
+from ....shared.patterns import (
+    KEYSEPARATOR,
+    POSITIVE_NUMBER_REGEX,
+    Collection,
+    FullQualifiedId,
+)
+from ....shared.schema import optional_id_schema
 from ...generics.update import UpdateAction
 from ...util.default_schema import DefaultSchema
 from ...util.register import register_action
 
+RERIDS_PATTERN = re.compile(r"\[(?P<fqid>\w+/\d+)\]")
```

Comment:
what is `RERIDS`? Please find a better name

Reply:
Replaced RERIDS_PATTERN with RECOMMENDATION_EXTENSION_REFERENCE_IDS_PATTERN.